pkg/kinflate/commands/inflate.go

```diff
@@ -87,8 +87,13 @@ func (o *inflateOptions) RunKinflate(out, errOut io.Writer) error {
 		return err
 	}
 
+	tGetter := tree.DefaultTransformer
+	if o.noop {
+		tGetter = tree.NoOpTransformer
+	}
+
 	// Do the transformation for the tree.
-	err = root.Inflate()
+	err = root.Inflate(tGetter)
```

Comment:
hmm. Can inflate take an bool arg noOp ? and hide this details from the caller ?

Reply:
That works for now.